### ISSUE TITLE
Learn how to deploy the application

### ISSUE THREAD

**Issue by bnbcfyh** [bnbcfyh/cmpe352-2021-repeat-project#9]
Should learn how to deploy a Docker app to AWS using Elastic Container Service so that I will be able to deploy this repeat app myself.

I am currently working on learning how to deploy the application since I have improved my knowledge on dockerizing the application. Therefore, I think I might be able to continue with the next step, deployment.

**Comment by bnbcfyh** [comment]
I found some good resources on the topic and followed the steps. However, highly likely since I am using an M1 Mac, I am experiencing problems with running my docker image on AWS. 

I read some recommendations on the topic and tried them but it didn't solve the problem. Now, I mostly focused on solving other issues.

**Comment by bnbcfyh** [comment]
I was finally able to create a docker image that is compatible with linux/amd64 on a different machine so that the compatibility issues were resolved.

After that, some issues still persisted and I checked the logs and found out that the problem I faced waS SQLAlchemy throwing an exception while trying to open a database. I referred to this [post](https://stackoverflow.com/questions/58175162/sqlalchemy-exc-operationalerror-sqlite3-operationalerror-unable-to-open-datab) and noticed that it might have been about the permissions. Therefore, I updated the [__init__.py](https://github.com/bnbcfyh/cmpe352-2021-repeat-project/blob/master/practice-app/website/__init__.py) file as well.

Also, I realized that there were some problems with my dockerization, hence, docker files are now updated as well.
